src/FortifyServiceProvider.php

```diff
@@ -99,6 +103,10 @@ public function registerComponents(): void
         Livewire::component('profile.update-profile-information-form', UpdateProfileInformationForm::class);
         Livewire::component('profile.update-profile-photo-form', UpdateProfilePhotoForm::class);
         Livewire::component('profile.update-timezone-form', UpdateTimezoneForm::class);
+        Livewire::component('auth.password-validator', PasswordValidator::class);
+        Livewire::component('auth.register-form', RegisterForm::class);
+
+        Blade::component('auth.password-rules', PasswordRules::class);
```

Comment:
Split `registerComponents` into `registerLivewireComponents` and `registerBladeComponents` so we can keep them clearly separated.